### ISSUE TITLE
Depthwise conv definition

### ISSUE THREAD

**Issue by EelcoHoogendoorn** [lucidrains/vit-pytorch#100]
https://github.com/lucidrains/vit-pytorch/blob/60b5687a7997f41c855ebc78ff77040ac5da5b61/vit_pytorch/pit.py#L92

The first layer in the depthwise conv should double the number of channels, as per fig 4 of the PiT paper; and it also makes sense to me to immediately double the channels; if not the first strided-but-equal-channel-conv will form an information bottleneck relative to the high res activation map upstream.

**Comment by lucidrains** [comment]
@EelcoHoogendoorn hmm, i do increase the channels at https://github.com/lucidrains/vit-pytorch/blob/60b5687a7997f41c855ebc78ff77040ac5da5b61/vit_pytorch/pit.py#L103

**Comment by EelcoHoogendoorn** [comment]
Yeah but its (dim_in, dim_in) in the first strided conv layer. Dont know if its a massive difference in practice, but its not according to the definition in the original paper.

**Comment by lucidrains** [comment]
@EelcoHoogendoorn oh got it! yea, i think there's a couple ways to do depthwise conv, fixing this!

**Comment by EelcoHoogendoorn** [comment]
Yeah not a depthwise-conv expert by any means; infact this was the first time i dug into the construct cause I got annoyed with the PiT paper not really explaining it imo; but if there are any differences left with the intended meaning in the PiT paper, there isnt any that I can spot.
